Description: Look in fallback pre-multiarch dir for external panels.
Author: Ryan Lortie <desrt@desrt.ca>
Origin: ubuntu
Forwarded: not-needed

diff -r -u gnome-control-center-3.6.3/shell/gnome-control-center.c gnome-control-center-3.6.3+awesome/shell/gnome-control-center.c
--- gnome-control-center-3.6.3/shell/gnome-control-center.c	2013-07-08 09:51:42.707462000 -0400
+++ gnome-control-center-3.6.3/shell/gnome-control-center.c	2013-07-08 09:52:37.723462000 -0400
@@ -904,6 +904,9 @@
   modules = g_io_modules_load_all_in_directory (PANELS_DIR);
   g_list_free (modules);
 
+  /* hardcoded fallback from pre-multiarching */
+  modules = g_io_modules_load_all_in_directory ("/usr/lib/control-center-1/panels");
+  g_list_free (modules);
 }
 
 
